extras/label_configs/carrots

```diff
@@ -1,5 +1,5 @@
 # Remove Github defaults
--import:clear-defaults
+-delete:all
```

Comment:
So unless we do this, there could be other labels that were setup previously that wouldn't be removed. It clears the edge case of something initializing with `carrots-ios` and then switching over to another label set.